src/current/_includes/releases/v24.1/v24.1.1.md

```diff
@@ -0,0 +1,103 @@
+- Reordered metrics on the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}) to improve their categorization. The metrics are now roughly in the following order: 1. Metrics to help determine which resource is constrained (IO, CPU); 2. Metrics to narrow down which [admission control]({% link {{ page.version.version }}/admission-control.md %}) queues are seeing requests waiting; 3. More advanced metrics about the system health (goroutine scheduler, L0 sublevels, etc.). [#124509][#124509]
+- Added the `cr.store.storage.l0-sublevels` and `cr.node.go.scheduler_latency-p99.9` metrics on the [**Overload** dashboard]({% link {{ page.version.version }}/ui-overload-dashboard.md %}) for better visibility into overloaded resources. [#124509][#124509]
+- There are now four separate graphs for Admission Queue Delay: 1. Foreground (regular) CPU work; 2. Store (IO) work; 3. Background (elastic) CPU work; 4. Replication Admission Control (store overload on replicas). [#124509][#124509]
+- The `Admission Queueing Delay – Store` chart now separates elastic (background) work from the regular foreground work. [#124509][#124509]
+- The `Admission IO Tokens Exhausted` chart now separates elastic and regular IO work. [#124509][#124509]
+
+<h3 id="v24-1-1-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that would occur when [`ALTER TYPE ... DROP VALUE`]({% link {{ page.version.version }}/alter-type.md %}) is followed by `DROP SCHEMA CASCADE ...` in the same transaction. Previously, the `ALTER TYPE` schema change would get queued up to run at commit time, but by that point, the type may have already been removed, so the commit could fail. [#123577][#123577]
+- CockroachDB now automatically repairs tables that see the error `invalid inbound foreign key ... origin table ID should be" or "invalid outbound foreign key ... reference table ID should be`. [#123668][#123668]
+- Fixed a bug where a failed [`RESTORE`]({% link {{ page.version.version }}/restore.md %}) could leave the system in a state where re-attempting the restore was not possible without manual intervention. [#123205][#123205]
+- Fixed a bug that could cause calling a routine to return an unexpected `function ... does not exist` error. The bug is triggered when the routine is called twice using the exact same SQL query, and either: (a) the routine has polymorphic arguments, or: (b) in between the two calls, the routine is replaced by a routine with the same name and different parameters. This bug has existed since alpha versions of v23.1. [#123516][#123516]
+- Fixed a rare bug where a lease transfer could lead to a `side-transport update saw closed timestamp regression` panic. The bug could occur when a node was overloaded and [failing to heartbeat its node liveness record]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues). [#123533][#123533]
```

Comment:
```suggestion
- Fixed a rare bug where a lease transfer could lead to a `side-transport update saw closed timestamp regression` panic. The bug could occur when a node was overloaded and [failing to heartbeat its node liveness record]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#node-liveness-issues). [#123533][#123533]
```
Would the panic occur because the previous node isn't reachable or the new node isn't reachable?

Reply:
I have no clue after re-reading the original PR, and am also not sure how much that question applies, so I'll avoid elaborating here.